packages/api/src/tasks/checkAlerts.ts

```diff
@@ -494,7 +494,7 @@ export const renderAlertTemplate = async ({
       displayType: DisplayType.Search,
       dateRange: [startTime, endTime],
       from: source.from,
-      select: savedSearch.select ?? source.defaultTableSelectExpression,
+      select: savedSearch.select || source.defaultTableSelectExpression || '*',
```

Comment:
The issue is  `??` only works for `null` or `undefined`.
```
const string = '' ?? 'testString';
// string: ''
```
Fix it by using `||`, and set default `*` if there is nothing select.

Reply:
ah that's great, imo we should not fall back to '*' here but the `||` fix is great

Reply:
yeah, most likely it will stop at `defaultTableSelectExpression`, `*` is for typescript complain(have change to be undefined).

Reply:
@MikeShi42 I can also replace * with '', so it will only remove alert body if user does not have `select` and `defaultTableSelectExpression `, which should not be happen.